app/views/reports/instrument_unavailable_reports/_report_tabs.html.haml

```diff
@@ -0,0 +1,13 @@
+%ul
```

Comment:
Seems like this duplicates `reports/instrument_reports/report_tabs`. Instrument day reports just do a render on that partial.